rpcs3/Emu/Cell/Modules/cellSsl.cpp

```diff
@@ -52,11 +60,15 @@ s32 cellSslCertificateLoader(u64 flag, vm::ptr<char> buffer, u32 size, vm::ptr<u
 	if (required)
 	{
 		*required = (u32)flagBits.count();
-		for (int i = 1; i < flagBits.size(); i++)
+		for (int i = 1; i <= flagBits.size(); i++)
 		{
 			if (!flagBits[i-1])
 				continue;
-			*required += (u32)(getCert(certPath, i).size() - 1);
+			// If we're loading cert 6 (the baltimore cert), then we need set that we're loading the 'normal' set of certs.
+			if (flagBits[5])
```

Comment:
This `if` looks fishy. Are you sure it should be run on every iteration? The condition doesn't change. Maybe factor it outside of the loop.

Reply:
Done